commands/publish/index.js

```diff
@@ -101,6 +101,20 @@ class PublishCommand extends Command {
       registry: this.options.registry,
     });
 
+    if (this.options["registry-scope"]) {
```

Comment:
Yargs automatically camelCases option names, so you can use `this.options.registryScope`.